cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/Python.kt

```diff
@@ -169,6 +177,8 @@ interface Python {
         override val returns: ASTBASEexpr? by lazy { "returns" of pyObject }
 
         override val type_comment: String? by lazy { "type_comment" of pyObject }
+
+        override var isAsync: Boolean = false
```

Comment:
This is redundant information. We already know this because `ast` returns a `ast.FunctionDef` instead of an `ast.AsyncFunctionDef`

Reply:
replaced with an interface.